Challenge 44&45: Vault Metadata challenge (#1147)
OWASP/wrongsecrets: src/main/resources/explanations/challenge44_reason.adoc

*Why putting sensitive data as metadata is a bad idea*

> Sometimes people reason that less sensitive data should be stored as metadata of an actual secret. Think of for instance having a username to be less sensitive as a password, but is it?
> In many of these cases these secrets are equally important and should get equal protection as the secret (e.g. the password) itself.
> 
> Very often we don't want to give read access to secrets to our employees, but we do want to give read access to metadata instead. If any secret is then stored in the metadata, that secret is then compromised internally.

Reviewer: ```suggestion
Sometimes, people reason that less sensitive data should be stored as secret metadata. Think of, for instance, a username - less sensitive than a password, or is it?
In many of these cases, these are equally important and should get equal protection as the secret (e.g. the password) itself.

We often don't want to give read access to secrets to our employees, but we do want to give read access to metadata instead. If any secret is stored in the metadata, that secret is then compromised internally.
```
